Plugins/GitUIPluginInterfaces/Settings/ChoiceSetting.cs

```diff
@@ -50,11 +50,11 @@ public override ComboBox CreateControl()
 
             public override void LoadSetting(ISettingsSource settings, ComboBox control)
             {
-                string settingVal = settings.SettingLevel == SettingLevel.Effective
+                string? settingVal = settings.SettingLevel == SettingLevel.Effective
                     ? Setting.ValueOrDefault(settings)
                     : Setting[settings];
 
-                control.SelectedIndex = Setting.Values.IndexOf(settingVal);
+                control.SelectedIndex = settingVal is null ? -1 : Setting.Values.IndexOf(settingVal);
```

Comment:
Bug fix. `IndexOf(null)` will throw.

Reply:
Please move it own commit as well

Reply:
4c4a428aa87fcee9c0687e84ca929321e085c0bc